zio-kafka-test/src/test/scala/zio/kafka/OOMSpecXmx300m.scala

```diff
@@ -0,0 +1,55 @@
+package zio.kafka
+
+import org.apache.kafka.clients.producer.ProducerRecord
+import zio.kafka.consumer.{ Consumer, Subscription }
+import zio.kafka.embedded.Kafka
+import zio.kafka.producer.Producer
+import zio.kafka.serde.Serde
+import zio.test.TestAspect._
+import zio.test._
+
+object OOMSpecXmx300m extends ZIOSpecWithSslKafka {
```

Comment:
🌕 Can you please put a class comment here explaining what issue these tests are testing, please?

Reply:
can I provide only the link to the original issue?

Reply:
Ok

Reply:
Fixed